code/mob/living/intangible/blob_overmind.dm

```diff
@@ -353,9 +354,10 @@
 		src.upgrade_id = 1
 		src.lipids = new()
 		src.nuclei = new()
-		src.my_material = copyMaterial(getMaterial("blob"))
-		src.my_material.color = "#ffffff"
-		src.initial_material = copyMaterial(getMaterial("blob"))
+		src.my_material = getMaterial("blob")
+		src.my_material = src.my_material.getMutable()
```

Comment:
Early in review, but this pattern seems to happen often enough that it might be worth a `getMutableMaterial` proc, for when you're starting from scratch and don't already have a material instance to work off of. Any reason not to do that?

Reply:
mostly because I don't want people to copy paste the pattern unless they **really** need a mutable material. If `getMutableMaterial()` exists, people will just cargo-cult it, and then we'll have to try and catch it in review instead. Explicitly making a material mutable mollifies that somewhat, and at least requires an extra proc call to be intentionally added. I also considered `getMaterial(material, mutable=FALSE)`  and discarded it for the same reason.

Reply:
Seems reasonable. Would appreciate a comment on getMutable explaining why it is that way (if there already is something like that, just ignore)